index.d.ts

```diff
@@ -104,6 +217,39 @@ declare class Emittery<
 	*/
 	static readonly listenerRemoved: typeof listenerRemoved;
 
+	/**
+	Debugging options for the current instance
+
+	@default
+	```
+	{
+		name: '',
+		enabled: false,
+		logger: (type, debugName, eventName, eventData) => {
+			if (typeof eventData === 'object') {
+				eventData = JSON.stringify(eventData);
+			}
+
+			if (typeof eventName === 'symbol') {
+				eventName = eventName.toString();
+			}
+
+			const currentTime = new Date();
+			const logTime = `${currentTime.getHours()}:${currentTime.getMinutes()}:${currentTime.getSeconds()}.${currentTime.getMilliseconds()}`;
+			console.log(`[${logTime}][emittery:${type}][${debugName}] Event Name: ${eventName}\n\tdata: ${eventData}`);
+		}
+	}
+	```
+	*/
```

Comment:
This `@default` block here is not needed. It's already document on the individual options.